src/Ocelot/Configuration/Validator/FileConfigurationFluentValidator.cs

```diff
@@ -101,6 +109,24 @@ private static bool IsPlaceholderNotDuplicatedIn(string upstreamPathTemplate)
             var upstreamPathPlaceholders = matches.Select(m => m.Value);
             return upstreamPathPlaceholders.Count() == upstreamPathPlaceholders.Distinct().Count();
         }
+
+        private static bool SameNamePlaceholdersInUpstreamandDownstream(FileRoute route)
```

Comment:
```suggestion
        private static bool IsPlaceholderDefinedInBothTemplates(FileRoute route)
```

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1927/commits/b27ea36ff193d2ba40c13943991fa2053b1aa6e5 ✔️